composer.json

```diff
@@ -6,7 +6,7 @@
     "license": "MIT",
     "autoload": {
         "psr-4": {
-            "Interop\\Container\\ServiceProvider\\": "src/"
+            "Interop\\Container\\ServiceProvider": "src/"
```

Comment:
It seems this is invalid Composer configuration (just received an email from Packagist). I wonder if it's worth dropping the PR entirely or using `Interop\Container` as the namespace?